vue3_ts/src/types/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './general.types'
 
 export * from '@/views/example-view/example-view.types'
 export * from '@/views/auth/auth.types'
+export * from '@/views/veevalidate-yup/veevalidate-yup.types'
```

Comment:
I think enums can be placed inside of veevalidate-yup.types other than 2 separate files, wdyt @m-tarabtsov @v-romanyuk ?

Reply:
Yes, as commented above, please place that enum into @/views/veevalidate-yup/veevalidate-yup.types.ts'